couchable/core.py

```diff
@@ -1580,22 +1655,23 @@ def findBadJson(obj, prefix=''):
             bad_list.extend(findBadJson(v, '{}[{}]'.format(prefix, i)))
     elif isinstance(obj, dict):
         for k, v in obj.items():
-            if isinstance(k, (str, int, float)):
+            if isinstance(k, string_types + integer_types + (float,)):
                 bad_list.extend(findBadJson(v, '{}[{!r}]'.format(prefix, k)))
             else:
                 bad_list.append('{} key {}'.format(prefix, k))
     elif isinstance(obj, float):
         if math.isnan(obj) or math.isinf(obj):
             bad_list.append('{} value {}, {}'.format(prefix, obj, type(obj)))
-    elif isinstance(obj, str):
+    elif isinstance(obj, bytes):
+        # By JSON standard there should only be unicode strings, but we keep this check
         try:
-            obj.encode('ascii')
```

Comment:
I have no idea how this part worked in the first place: you can't encode a byte string, a byte string is already encoded, you can only decode a byte string.